site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,403 @@
+			<p>Tab Item 5</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-one-6" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+</tab-container>
+```
+
+## Segmented Control with footer Next/Prev
+
+Removing the next and previous buttons will still provide a seamless experience on the tabs control. Small restructuring allows for the tabs-container component to allocate a footer to show after the tabpanel. The key difference is that of the `div.segmented-controls` taking on the `role=tablist` and `aria-orientation` attributes. This restructuring provides the grid responsiveness without buttons next to the control-list.
+
+```html
+<tab-container data-segmented-controls-container>
+	<div class="segmented-controls" role="tablist" aria-orientation="horizontal">
+		<div class="segmented-control-list">
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-1"
+				role="tab"
+				aria-selected="true"
+				data-segmented-control
+			>
+				Define Strategy
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-2"
+				role="tab"
+				data-segmented-control
+			>
+				Plan
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-3"
+				role="tab"
+				data-segmented-control
+			>
+				Prepare
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-4"
+				role="tab"
+				data-segmented-control
+			>
+				Adopt
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-5"
+				role="tab"
+				data-segmented-control
+			>
+				Govern
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-6"
+				role="tab"
+				data-segmented-control
+			>
+				Manage
+			</button>
+		</div>
+	</div>
+
+	<div role="tabpanel" aria-labelledby="sc-three-1">
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 1</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-2" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 2</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-3" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 3</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-4" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 4</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-5" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 5</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-6" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+
+	<!-- After Tabpanels -->
+	<div class="segmented-controls-footer border-top display-flex padding-top-sm">
+		<button
+			class="button button-sm button-primary margin-right-sm"
+			title="Previous segment"
```

Comment:
Is `title` needed here? Same question for the next button. These two aren't icon only button so probably should be removed.

Reply:
**I'd be in favor of this.** A speech control user would, based on the visual text, expect to say "click Previous" to click the "Previous"/"Previous segment" button, but `title` could make it so that some speech control software is specifically expecting "click Previous segment," with no clear visual indicator of that expectation. If we're showing some visual text, it's better to not also have hidden text that could sway the accessible name, to make targeting this button more predictable.